app/frontend/src/components/Team.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <v-container>
+    <p class="text-right">
+      <strong>
+        <v-icon class="pr-2">group</v-icon>Team members
+        <span v-if="usersLoaded">({{users.length}})</span>
+        <v-tooltip bottom open-delay="1000">
+          <template v-slot:activator="{ on }">
+            <v-btn v-on="on" icon color="primary" @click="addDialog = true">
+              <v-icon>add_circle</v-icon>
+            </v-btn>
+          </template>
+          <span>Invite a new user to your team</span>
+        </v-tooltip>
+      </strong>
+    </p>
+    <v-skeleton-loader
+      type="table"
+      :types="{'table': 'table-thead, table-tbody', 'table-row': 'table-cell@3', 'table-thead': 'heading@3' }"
+      :loading="!usersLoaded"
+      transition="scale-transition"
+    >
+      <v-container>
+        <v-alert
+          v-if="errorLoadingUsers"
+          type="error"
+          tile
+          dense
+        >An error occurred while loading the users, please refresh and try again</v-alert>
+        <v-simple-table class="getok-simple-table">
+          <template v-slot:default>
+            <thead>
+              <tr>
+                <th class="text-left">Name</th>
+                <th class="text-left">IDIR</th>
+                <th class="text-left">Email</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr v-for="item in users" :key="item.idir">
+                <td>{{ item.name }}</td>
+                <td>{{ item.idir }}</td>
+                <td>{{ item.email }}</td>
+              </tr>
+            </tbody>
+          </template>
+        </v-simple-table>
+      </v-container>
+    </v-skeleton-loader>
+
+    <BaseDialog :show="addDialog" width="500" @close-dialog="addDialog = false">
```

Comment:
Width doesn't need to be specified here as it's already a prop default

Reply:
Fixed